setup.go

```diff
@@ -34,6 +34,8 @@ func setup(c *caddy.Controller) error {
 	m := MDNS{Domain: domain, mutex: &mutex, mdnsHosts: &mdnsHosts, srvHosts: &srvHosts}
 
 	c.OnStartup(func() error {
+		// mdns is quite noisy, and we don't really care about any of its messages
+		mdns.DisableLogging = true
```

Comment:
Shouldn't we check if the debug plugin is enabled and leave it enabled in that case?

Reply:
Maybe? None of the mdns log messages I've seen so far have been useful and they just clutter up the logs. I suppose it's possible that at some point we'd have an mdns-related problem and need them though.

Reply:
I went ahead and left the mdns logging enabled when debug is turned on. It's still noisy, but so is our debug logging so it won't hurt much.